factory/runType/sovereignRunTypeComponents.go

```diff
@@ -202,7 +196,11 @@ func (rcf *sovereignRunTypeComponentsFactory) Create() (*runTypeComponents, erro
 
 	expiryTime := time.Second * time.Duration(rcf.sovConfig.OutgoingSubscribedEvents.TimeToWaitForUnconfirmedOutGoingOperationInSeconds)
 
-	err = rtc.extraHeaderSigVerifierHandler.RegisterExtraHeaderSigVerifier(rcf.extraVerifier)
+	sovHeaderSigVerifier, err := headerCheck.NewSovereignHeaderSigVerifier(rcf.cryptoComponents.BlockSigner())
```

Comment:
Is this also removed from normal run type ?

Reply:
Normal RunType doesn't use an extra header sig verifier, it's just:
```extraHeaderSigVerifierHandler:           headerCheck.NewExtraHeaderSigVerifierHolder(),```

Reply:
The name of the variable was wrong and confused me;
extraHeaderSigVerifierHandler -> this is a holder, not a handler, please rename it